wizeline/remix-project-lab
# Feat/209/internship projects

# What does this PR do?


This pr add an interns page for users with an applicant role, this means that they log in with LinkedIn, an they can apply to projects that have mentors, also this pr implements the logic to redirect depending on the role that the user have.

# Where should the reviewer start?


app/routes/internshipProjects

# How should this be manually tested?


You need to delete all your cookies, then yo log in with LinkedIn, first you are going to be redirect to /applicationform, after you complete the form you will be redirect to /internshipPorjects, you only have access to some projects, if you click on one of them you will see some description of the project an a Botton that says apply

IMPORTANT: Apply Botton doesn't work yet, @AxelUZ is going to make it work

# Any background context you want to provide?



# What are the relevant tickets?


#209 

# Screenshots



![Captura de pantalla 2023-09-18 a la(s) 13 19 40](https://github.com/wizeline/remix-project-lab/assets/93209499/31703034-df1e-4c35-91db-a816eba3c7b2)

![Captura de pantalla 2023-09-18 a la(s) 13 22 04](https://github.com/wizeline/remix-project-lab/assets/93209499/ed4e9d50-4ff5-4ce5-9eb4-8e03b1b729cc)

![Captura de pantalla 2023-09-18 a la(s) 13 22 34](https://github.com/wizeline/remix-project-lab/assets/93209499/a10c2625-ca92-4443-b103-0132830d7c15)

![Captura de pantalla 2023-09-18 a la(s) 13 22 55](https://github.com/wizeline/remix-project-lab/assets/93209499/99d27b73-af22-4b18-b8b4-44999142dd3a)

![Captura de pantalla 2023-09-18 a la(s) 13 24 43](https://github.com/wizeline/remix-project-lab/assets/93209499/83ed341c-4595-4734-beba-6e78c47e8fe9)

![Captura de pantalla 2023-09-18 a la(s) 13 25 05](https://github.com/wizeline/remix-project-lab/assets/93209499/6b241240-ffd9-4c26-a76d-86674c4ac06d)

![Captura de pantalla 2023-09-18 a la(s) 13 25 29](https://github.com/wizeline/remix-project-lab/assets/93209499/48fa8140-c51e-423b-83d5-2d9f5226a098)


# Questions



# Checklist
